### PR TITLE
13.2.0

### PR DESCRIPTION
**Pull request by agliga**
Merge to master for release

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,23 +74,23 @@
         "@commitlint/cli": "^18",
         "@commitlint/config-conventional": "^18",
         "@ebay/browserslist-config": "^2.7.0",
-        "@ebay/skin": "~17.1.0",
+        "@ebay/skin": "~17.2.0-next.0",
```
Comment (ArtBlue) [inline package.json:77]: Should this be changed to the actual release or will changeset do that automatically during the release?

File: docs/assets/getting-started.stories-fKLBwFDx.js
Comment (ArtBlue) [inline docs/assets/getting-started.stories-fKLBwFDx.js:1]: What's up with all these auto-generated docs `js` files?

**Review by ArtBlue — COMMENTED** [review]
Looks good. Just a couple of questions and a possible Skin version change.

File: docs/assets/getting-started.stories-fKLBwFDx.js
Comment (agliga) [reply docs/assets/getting-started.stories-fKLBwFDx.js:1]: Those are part of the build and needed for release. 
Usually those are part of the commit that we push up to master but they ended up being part of this prerelease commit

File: package.json (hunk 1 above)
Comment (agliga) [reply package.json:77]: I'll swap it after skin release

**Review by ArtBlue — APPROVED** [review]

**Pull request merged**